pagerduty_handler.json

```diff
@@ -0,0 +1,8 @@
+{
+  "handlers": {
+    "pagerduty": {
+      "type": "pipe",
+      "command": "/usr/share/pdagent-integrations/bin/pd-sensu -k YOUR-INTEGRATION-KEY-HERE"
+    }
+  }
+}
```

Comment:
I chose `conf.d` as that is a fairly common name, and I added `sensu_` to the filename to avoid ambiguity / potential name collision.